microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md

```diff
@@ -177,6 +175,9 @@ Points to address or value:	selector2-cohowinery-com._domainkey.cohovineyardandw
 TTL:				3600
 ```
 
+> [!NOTE]
+> Creating the second record is important to do but only one of the selectors may be available at the time of creation. In essence, the second selector may point to an address that has not been created yet. It is still recommended that the second CNAME record is created since it will come in handy when the keys rotate and, as an admin, there would be nothing to do and the rotation would be seamless.
```

Comment:
```suggestion
> It is important to create the second record but only one of the selectors may be available at the time of creation. In essence, the second selector may point to an address that has not been created yet. It is still recommended that the second CNAME record is created since it will come in handy when the keys rotate and, as an admin, there would be nothing more to do and the rotation would be seamless.
```